.golangci.example.yml

```diff
@@ -590,6 +590,10 @@ linters-settings:
     # tab width in spaces. Default to 1.
     tab-width: 1
 
+  maintidx:
+    # show functions with maintainability index < N only.
+    under: 20
```

Comment:
```suggestion
    # Show functions with maintainability index lower than N.
    # A high index indicates better maintainability (it's kind of the opposite of complexity).
    # default: 20
    under: 100
```

Reply:
I see, you are right, thanks. I've added it.